plugins/exporter.koplugin/clip.lua

```diff
@@ -238,8 +246,11 @@ function MyClipping:parseHighlight(highlights, bookmarks, book)
             clipping.chapter = item.chapter
             for _, bookmark in pairs(bookmarks) do
                 if bookmark.datetime == item.datetime and bookmark.text then
-                    local tmp = string.gsub(bookmark.text, "Page %d+ ", "")
-                    clipping.text = string.gsub(tmp, " @ %d%d%d%d%-%d%d%-%d%d %d%d:%d%d:%d%d", "")
+                    local tmp = string.gsub(bookmark.text, "^Page %[?%d*%]?%d+ ", "")
```

Comment:
This feels like a bug, albeit presumably due to being copied from a bit of very old fossilized code.

Reply:
Are you concerned about the pattern? Or something else? Any preference on how I should change this?

Reply:
Either "Page" should be translated and this works by accident but shouldn't, or it's already broken.

Reply:
https://github.com/koreader/koreader/blob/8f1d219154e09e1e7d4bfee867a9611f3fc647fb/frontend/apps/reader/modules/readerbookmark.lua#L1147

Reply:
Hm, so the issue is that it's taking (or can only take?) automatically created text as input without really knowing what was at the basis of it?

Reply:
Yeah. I was initially trying to keep the original behavior here, but maybe I should get rid of this behavior instead. I could instead just check if `bookmark.text` is auto-text with `isBookmarkAutoText`. If it's not auto-text, I'll put it the whole thing (without any `string.gsub()`) into into `clipping.note`.

This way we're not trying to interpret a generated and subsequently user-modified string.

Reply:
I'm a bit confused as to whether you're saying it's an ancient fossil that you copied or not?

Reply:
No, I didn't copy this from anywhere else. The original code here used the pattern "Page %d+ "

I modified it to also catch the page format from non-linear flows (i.e. "Page [2]4 "), based on a request from @hius07 : https://github.com/koreader/koreader/pull/8548/files/669660c1aac6b3a363dce52bfe61042ac6896967#diff-c9c35c46d3174f706b0ae587687fdfca0c6de2ed12af834b57740cac03fa27f1

The intent of the original code seems to be to allow modifying the highlight auto-text, but automatically stripping off the page # and time to avoid redundant exported info. If the text doesn't contain the page number at the start or the date/time at the end, it will just take the full string.

IMO, we should just remove this behavior since it's complicated to know if the user is just typing a comment like "Page 5 is very interesting"

Reply:
Well, that does make it an ancient fossil. But whatever you do, please keep it the same everywhere. (Preferably get rid of it completely.)